File: second-semester/Data-structure-and-algorithms-I/workshops/seguimiento-0/condicionales/helado3.py
temperatura_actual = int(input())
edad = int(input())

if temperatura_actual > 27 and edad >= 18:
    dinero = int(input())
    if dinero > 5:
        print("Comprar helado cerveza")
    else:
        print("Lo sentimos estas pobre")
else:
    print("Lo sentimos juventud")

File: 1-semester/Programming-Fundamentals/labs/Conditionals/Helado3.java
import java.util.Scanner;
public class Main {
  public static void main(String[] args) {
    Scanner texto = new Scanner(System.in);
    int temperatura = texto.nextInt();
    int edad = texto.nextInt();
    if (temperatura > 27 && edad >= 18) {
      int dinero = texto.nextInt();
      if (dinero > 5000) {
        System.out.println("Comprar helado cerveza");
      }
      else {
        System.out.println("De que me hablas viejo");
      }
    }
    else {
      System.out.print("Lo sentimos juventud");
    }
  }
}